doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+     If it does not contain a PIN, enter the Hardware ID (HWID) HEX code, with or without colons.
+     For example, *AA:BB:CC:DD:EE:FF* or *AABBCCDDEEFF*.
+
+     .. note::
+
+        The ownership code serves as a password and proves that you own the specific nRF9160 DK.
+        Therefore, you should not share it with anyone.
+
+#. Click :guilabel:`Add Device` and wait for the device to reconnect to nRF Cloud.
+   It is normal for the device to disconnect and reconnect multiple times during device provisioning.
+
+The nRF9160 DK is now added to your nRF Cloud account.
+This is indicated by the blinking of **LED3** on the DK, which shows that it is publishing data.
+See :ref:`Asset Tracker v2 LED indication <led_indication>` for more information.
+
+On nRF Cloud, you can access the device by clicking :guilabel:`Devices` under :guilabel:`Device Management` in the navigation pane on the left.
+
+.. _nrf9160_gs_testing_dk:
+
+Testing the DK
+**************
+
+After successfully associating your nRF9160 DK with your nRF Cloud account, you can start testing it.
+The application programmed in the DK is :ref:`asset_tracker_v2`, and it is used for the testing.
+
+For a basic test, complete the following steps:
+
+1. Turn on or reset your nRF9160 DK.
+#. Go to `nRF Cloud` and sign in.
+#. Click :guilabel:`Devices` under :guilabel:`Device Management` in the navigation pane on the left.
+
+   .. figure:: /images/nrfcloud_devices.png
+      :alt: nRF Cloud - Devices
+
+      nRF Cloud - Devices
+
+#. From the :guilabel:`Devices` view, open the entry for your device.
+#. Observe that the DK is sending data to nRF Cloud.
+
+If you experience problems and need to check the log messages, open nRF Connect for Desktop and launch the LTE Link Monitor app.
+After connecting to your DK, you can see the log messages in the terminal view.
+
+.. _nrf9160_gs_testing_cellular:
+
+Testing the cellular connection with the AT Client sample
+=========================================================
+
+The :ref:`at_client_sample` sample enables you to send AT commands to the modem of your nRF9160 DK to test and monitor the cellular connection.
+You can use it to troubleshoot and debug any connection issues.
+
+Complete the following steps to test the cellular connection using the AT Client sample:
+
+1. Follow the steps in :ref:`nrf9160_gs_updating_fw_application` to program the sample to the DK.
+   When choosing the HEX file, choose `nrf9160dk_at_client_<version-number>.hex` instead of one for Asset Tracker v2.
+#. Test the AT Client sample as described in the Testing section of the :ref:`at_client_sample` documentation.
+
+.. _ug_nrf9160_gs_testing_gnss:
+
+Testing the GNSS functionality
+==============================
+
+If supported by the network, :ref:`asset_tracker_v2` retrieves GNSS data and transmits it to nRF Cloud.
+
+The following conditions must be met so that the nRF9160 DK can retrieve GNSS data:
```

Comment:
I suggest to change this sentence and the section below:

```
To achieve the fastest Time To First Fix og GNSS position, the following conditions should be met:
* The device must be able to connect to nRF Cloud. You can confirm this by checking whether the status of your DK is displayed correctly on nRF Cloud. The cloud connection is used to download GPS assistance data.
* Your network operator should support Power Saving Mode (PSM) or Extended Discontinuous Reception (eDRX) with the SIM card that you are using.  If you are using an iBasis SIM card, check the `iBasis network coverage spreadsheet`_ to see the supported features and network coverage for different countries.

The device may be able to acquire a GNSS position fix also if the network does not support PSM or eDRX for your SIM, although it likely will take longer to do so.
```

Reply:
Changed.